mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergInputFormat.java

```diff
@@ -45,9 +47,9 @@
 import static org.apache.iceberg.types.Types.NestedField.required;
 
 @RunWith(StandaloneHiveRunner.class)
-public class TestHiveIcebergInputFormat {
+public abstract class TestHiveIcebergInputFormat {
```

Comment:
it is kinda weird that we are calling this class `TestHiveIcebergInputFormat` instead of say `TestHiveStorageHandler` or something similar.

Reply:
That's because this test class was added before the storage handler. I'll rename it.